### ISSUE TITLE
Documentation error: 

### ISSUE THREAD

**Issue by WaterSibilantFalling** [v1k45/python-qBittorrent#15]
The first example of
**Get all active torrents:**
is
qb.torrents()

This is incorrect. This returns only 10 torrents.
```
def torrents( self, status='active', label='', sort='priority',                     
                 reverse=False, limit=10, offset=0):   
```

Actually, this is a coding error. The default limit should be 85,000 or 65,336 or 99,999 or some clearly 'maximum ever' number. Currently, the limit must be inserted in every function call. Instead, it should only be required when it is ... required: when a limit is needed.

**Comment by v1k45** [comment]
@RichardWilliamPearse yeah, you are right, i was being conservative with the limit.

Setting default limit implicitly was not a good idea. It should be changed to the default number of torrents provided by qBitorrent's WEB API.

**Comment by v1k45** [comment]
I just realized that I unknowingly fixed it in this commit: 3b26cde. But this fix isn't updated on PyPi because the documentation was not ready.

**Comment by v1k45** [comment]
I've updated the PyPI project; you can update your package to use the latest version.
